pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -114,7 +131,7 @@ func TestDevicePluginReRegistration(t *testing.T) {
 	close(callbackChan)
 }
 
-func setup(t *testing.T, devs []*pluginapi.Device, callback monitorCallback) (Manager, *Stub) {
+func setup(t *testing.T, devs []*pluginapi.Device, callback monitorCallback, socketName string, pluginSocketName string) (Manager, *Stub) {
```

Comment:
same as above

Reply:
Is this comment perhaps attached to the wrong line? If it was related to ".TestDevicePluginReRegistration.third", then I fixed that.

If the comment really was about the setup() line, then I don't understand what I should change here.

Reply:
you got it right. comment on wrong line.